### ISSUE TITLE
Support for AWS CDK

### ISSUE THREAD

**Issue by vishnus17** [bregman-arie/devops-exercises#273]
CDK is growing rapidly so it'd be helpful to bring up a community support for it.

**Comment by bregman-arie** [comment]
Can you expand? Do you mean you think it would be good to see it used in the solutions for AWS exercises?

**Comment by vishnus17** [comment]
Yes. We could do that and provide solutions for the applicable exercises or provide community support by giving seperate exercises in CDK. I could help with that as I'm currently working on that.
